package.json

```diff
@@ -2,7 +2,9 @@
   "name": "mongosh",
   "description": "The MongoDB Shell",
   "main": "index.js",
-  "bin": "packages/cli-repl/bin/mongosh.js",
+  "bin": {
+    "mongosh": "packages/cli-repl/bin/mongosh.js"
+  },
```

Comment:
I think I accidentally used a newer npm version at some point that did this automatically :woman_shrugging: